processor/src/stack/mod.rs

```diff
@@ -77,7 +91,11 @@ impl Stack {
 
         if num_items > STACK_TOP_SIZE {
             let num_overflow_items = num_items - STACK_TOP_SIZE;
-            result.extend_from_slice(&self.overflow[..num_overflow_items]);
+            let prev_size = result.len();
+            self.overflow.add_values(&mut result);
```

Comment:
if we change `overflow.add_values()` to take the number of items to copy - this could be made much simpler.

Reply:
I think you asked me to remove it last comment

Reply:
fixed